otelx/jaeger.go

```diff
@@ -18,8 +18,8 @@ import (
 // Optionally, Config.Providers.Jaeger.LocalAgentAddress can be set.
 // NOTE: If Config.Providers.Jaeger.Sampling.ServerURL is not specfied,
 // AlwaysSample is used.
-func SetupJaeger(t *Tracer, tracerName string) (trace.Tracer, error) {
-	host, port, err := net.SplitHostPort(t.Config.Providers.Jaeger.LocalAgentAddress)
+func SetupJaeger(t *Tracer, tracerName string, c *Config) (trace.Tracer, error) {
```

Comment:
Does it make sense to pass 2 times almost the same object? Or can `t.Config` differ from `Config`?

Reply:
I've removed the `Config` field from the tracer, so we need to pass the config manually.